templates/web.letsencrypt.ssl.template.yml

```diff
@@ -0,0 +1,58 @@
+env:
+  LETSENCRYPT_DIR: "/shared/letsencrypt"
+
+hooks:
+  after_ssl:
+    - exec:
+       cmd:
+         - if [ -z "$LETSENCRYPT_ACCOUNT_EMAIL" ]; then echo "LETSENCRYPT_ACCOUNT_EMAIL ENV variable is required and has not been set."; exit 1; fi
+         - /bin/bash -c "if [[ ! \"$LETSENCRYPT_ACCOUNT_EMAIL\" =~ ([^@]+)@([^\.]+) ]]; then echo \"LETSENCRYPT_ACCOUNT_EMAIL is not a valid email address\"; exit 1; fi"
+
+    - exec:
+       cmd:
+         - apt-get install -y netcat
+         - cd /root && git clone https://github.com/Neilpang/acme.sh.git && cd /root/acme.sh && git reset --hard 8d5618c44a2ab973aa7eb243db740e22c742b809
```

Comment:
As I read here this will pull down a git repository and execute a shell script directly from a specific commit. Two issues I see here: pulling down & executing an ssh script from an external source and not having a versioned external library that we use for checking this out. What do you think about these issues?

Also, what is ACME? :)